codelists/tests/test_api.py

```diff
@@ -262,6 +263,34 @@ def test_codelists_get_all(client, organisation, organisation_user):
     assert {cl["user"] for cl in user_codelists} == {organisation_user.username}
 
 
+def test_codelists_get_all_still_works_for_a_codelist_with_a_missing_handle(
```

Comment:
thought: I quickly copied the existing `get_all` test; it's possible we can reduce this down a bit.